File: Java_Algo/프로그래머스/unrated/181943. 문자열 겹쳐쓰기/문자열 겹쳐쓰기.java
class Solution {
    public String solution(String my_string, String overwrite_string, int s) {
        String answer = "";
        
        int e = overwrite_string.length();
        char tmp;
            
        for (int i = 0; i < my_string.length(); i++) {
            if ((i >= s) && (i < e + s)) {
                tmp = overwrite_string.charAt(i - s);
                answer += tmp;
            } else {
                tmp = my_string.charAt(i);
                answer += tmp;
            } 
        }
        
        return answer;
    }
}

File: 프로그래머스/unrated/181943. 문자열 겹쳐쓰기/문자열 겹쳐쓰기.js
function solution(my_string, overwrite_string, s) {
    var answer = '';
    
    var x = s + overwrite_string.length
    
    my_string[s].replace(overwrite_string[0])
    
    for (var i = 0; i < x; i++) {
        if (i < s) {
            answer = answer + my_string[i]
        } else {
            answer = answer + overwrite_string
            
            break;
        }
    }
    
    if (my_string.length != x) {
        for (var k = x; k < my_string.length; k++) {
            answer = answer + my_string[k]
        }
    }

    return answer;
}